test/navigation/OtherOfficialsNavigatorSpec.scala

```diff
@@ -570,12 +623,23 @@ class OtherOfficialsNavigatorSpec extends SpecBase {
                 routes.SessionExpiredController.onPageLoad()
             }
 
+            "go to the check your answers page when yes and prev address is defined is selected" in {
+              navigator.nextPage(IsOtherOfficialsPreviousAddressPage(index), PlaybackMode,
+                emptyUserAnswers.set(IsOtherOfficialsPreviousAddressPage(0), true)
+                  .flatMap(_.set(OtherOfficialPreviousAddressLookupPage(0), address))
+                  .flatMap(_.set(OtherOfficialPreviousAddressLookupPage(previousOrSameIndex(index)), address))
+                  .flatMap(_.set(OtherOfficialPreviousAddressLookupPage(index), address))
+                  .flatMap(_.set(IsOtherOfficialsPreviousAddressPage(previousOrSameIndex(index)), true))
```

Comment:
make those lines in order (ie group them by what page you're setting). will be ready to merge afterwards.